### PR TITLE
Add dtisaac01 keyboard

### PR DESCRIPTION
**Pull request by daotakisaac**
Added a new base keyboard compatible with QMK Firmware.

## Description

I have developed a new keyboard. This PR contains all the files needed to build the default keymap of this keyboard.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/dtisaac01/config.h [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+#define BACKLIGHT_LEVELS 3
+#define BACKLIGHT_BREATHING
+
+/* Debounce reduces chatter (unintended double-presses) - set 0 if debouncing is not needed */
+#define DEBOUNCE 5
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
```
Comment (drashna) [inline keyboards/handwired/dtisaac01/config.h:65]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/handwired/dtisaac01/readme.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# dtisaac01
+
+![dtisaac01](imgur.com image replace me!)
+
+*A short description of the keyboard/dtisaac01
```
Comment (fauxpark) [inline keyboards/handwired/dtisaac01/readme.md:5]: Please fill these out.

File: keyboards/handwired/dtisaac01/info.json [hunk 3]
```diff
@@ -0,0 +1,101 @@
+{
+    "keyboard_name": "dtisaac01",
+    "url": "",
+    "maintainer": "DTIsaac",
+    "width": 18.25,
+    "height": 6.5,
+    "layouts": {
+        "LAYOUT": {
+			"key_count": 87,
```
Comment (fauxpark) [inline keyboards/handwired/dtisaac01/info.json:9]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/handwired/dtisaac01/readme.md (hunk 2 above)
Comment (daotakisaac) [reply keyboards/handwired/dtisaac01/readme.md]: I filled these in the new patch set.
Please help me check again and merge it.
Thanks.

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**